horizons/ai/aiplayer/building/farm.py

```diff
@@ -263,8 +263,12 @@ def init_field_offsets(cls):
 		cls.__field_offsets = first_class + second_class + third_class
 
 	@classmethod
-	def _suitable_for_road(self, production_builder, coords):
-		return coords in production_builder.land_manager.roads or (coords in production_builder.plan and production_builder.plan[coords][0] == BUILDING_PURPOSE.NONE)
+	def _suitable_for_road(cls, production_builder, coords):
+		"""check coordinates
+		returns true/false"""
```

Comment:
If that comment is really necessary (in my opinion, its not), then it should bue `True/False`.